### DISCUSSION TITLE
questions about cmd report generation

### DISCUSSION THREAD

**Discussion by zemaria523** [sepinf-inc/IPED#1129] [Q&A] [answered]
My case:
We process the evidence with IPED, make the bookmarks, and generate a new report only with those bookmarks.

After, we put this new report in other computer and make new bookmarks from the preview ones.  

My questions:
- Is there a way, from commandline, to generate a new iped report with those new bookmarks? with iped jar from the evidence (as I can from GUI)
- Is there a way to set those parameters: examiner, date, case (from the html report)... etc in commandline? 
- Is there a way to set true the html report in commandline? We usually set it as false in config file.

**Comment by lfcnassif** [comment, marked as answer]
> * Is there a way, from commandline, to generate a new iped report with those new bookmarks?

It should work if you download and use the same iped version used to generate the report. Other option with 3.18.x version is to run `java -cp indexador/lib/iped-search-app.jar dpf.sp.gpinf.indexer.IndexFiles -d marcadores.iped -o output`


> * Is there a way to set those parameters: examiner, date, case (from the html report)... etc in commandline?

You can create a json file with those params and pass it to -asap cmd line option. This is the json syntax:
https://github.com/sepinf-inc/IPED/wiki/User-Manual#report-creation



> * Is there a way to set true the html report in commandline?

There isn't, you should change it in IPEDConfig.txt

**Comment by zemaria523** [comment]
Thank you and sorry, didnt see the discussion tab.
